### DISCUSSION TITLE
OreintDB Roadmap

### DISCUSSION THREAD

**Discussion by hetawfik** [orientechnologies/orientdb#9999] [General]
I was wondering on OrientDB road map, is OrientDB still an active open source project? and if so what's the road map for future releases?

**Comment by lvca** [comment]
I think the lack of an answer speaks for itself :-) As you can see there has not much activity on the OrientDB project for a long time and no company is actively sponsoring the project. Please consider using ArcadeDB as the closest project (born as a conceptual fork from OrientDB) or if you need "only" a graphdb checkout Neo4j. If you mostly need a document database, MongoDB could be a good fit.
